src/praisonai-agents/test_agent_guardrails.py

```
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+Simple test for Agent guardrail functionality.
+"""
+
+def test_agent_guardrails():
+    """Test basic Agent guardrail functionality."""
+    print("Testing Agent guardrail functionality...")
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)